lib/jsonapi_errors_handler/errors/standard_error.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+require_relative '../../keys_stringifier'
```

Comment:
This probably can be solved with `require 'jsonapi_errors_handler/keys_stringifier` after moving the `keys_stringifier` into a `jsonapi_errors_handler` namespace